public/css/ux_style.css

```diff
@@ -43,6 +43,11 @@ body {
     background: var(--button);
 }
 
+.chore-form input[type="submit"]:disabled {
```

Comment:
I don't see this being hit. Let me know if you want to talk through it.

Reply:
In css was using . instead should have been using # to get to the right element.